packages/builds/src/node/error-parser.spec.ts

```diff
@@ -127,9 +127,11 @@ describe("error-parser", () => {
             const readStream: NodeJS.ReadableStream = fs.createReadStream(__dirname + logName);
             const promise = parser.parse(gccLinkerErrorMatcher, readStream);
 
+            // TODO: Fix on windows:
+            // "file": "C:\\tmp\\gdb-8.0\\zlib\\inflate.c"
             return expect(promise).to.eventually.deep.equal(
                 [{
-                    "file": "/tmp/gdb-8.0/zlib/inflate.c",
+                    "file": path.resolve("/tmp/gdb-8.0/zlib/inflate.c"),
```

Comment:
Thanks - i was not aware we have our own version of Path. After thinking about it a bit, I think it's easier, while in the back-end back-end, to use the more feature-rich node path. When it comes time to send things to the front-end (e.g. error markers), we can convert the paths to Theia's normalized version.